{
    "title" : "\"Request Error : Error while performing JWT flow for the requested user\"",
    "uri" : "http://answers.coveo.com/questions/10855/request-error-error-while-performing-jwt-flow-for.html",
    "printableUri" : "https://answers.coveo.com/questions/10855/request-error-error-while-performing-jwt-flow-for.html",
    "clickUri" : "https://answers.coveo.com/questions/10855/request-error-error-while-performing-jwt-flow-for.html",
    "uniqueId" : "42.4247$http://answers.coveo.com/questions/10855/request-error-error-while-performing-jwt-flow-for.html",
    "excerpt" : "I'm getting this error when accessing my community as a sysadmin user: ... I've already looked at the 2 following questions: ... However the suggested answers don't help: I've already made sure tha...",
    "firstSentences" : null,
    "summary" : null,
    "flags" : "HasHtmlVersion",
    "hasHtmlVersion" : true,
    "hasMobileHtmlVersion" : false,
    "score" : 1176,
    "percentScore" : 77.164955,
    "rankingInfo" : null,
    "rating" : 3.0,
    "isTopResult" : false,
    "isRecommendation" : false,
    "titleHighlights" : [ ],
    "firstSentencesHighlights" : [ ],
    "excerptHighlights" : [ ],
    "printableUriHighlights" : [ ],
    "summaryHighlights" : [ ],
    "parentResult" : null,
    "childResults" : [ ],
    "totalNumberOfChildResults" : 0,
    "raw" : {
      "systitle" : "\"Request Error : Error while performing JWT flow for the requested user\"",
      "sysauthor" : "laurent",
      "sysurihash" : "E9kVLMÒdfvk1WQÒJ",
      "urihash" : "E9kVLMÒdfvk1WQÒJ",
      "numberofanswers" : 0,
      "sysuri" : "https://answers.coveo.com/questions/10855/request-error-error-while-performing-jwt-flow-for.html",
      "sysprintableuri" : "https://answers.coveo.com/questions/10855/request-error-error-while-performing-jwt-flow-for.html",
      "sitemappriority" : 0.1,
      "systransactionid" : 160364,
      "questionaskeddatestring" : "∑\n            Mar 23, 2017 at 11:43 AM ∑",
      "tags" : [ "coveo express" ],
      "sysconcepts" : "sysadmin user ; community ; admin identity ; connected app ; org ; visualforce ; fallback ; administrator ; original post ; laurent gminero ; Oauth Policies ; sensitive data ; credentials offline ; permission sets ; suggested answers",
      "concepts" : "sysadmin user ; community ; admin identity ; connected app ; org ; visualforce ; fallback ; administrator ; original post ; laurent gminero ; Oauth Policies ; sensitive data ; credentials offline ; permission sets ; suggested answers",
      "printableuri" : "https://answers.coveo.com/questions/10855/request-error-error-while-performing-jwt-flow-for.html",
      "documenttype" : "WebPage",
      "sysindexeddate" : 1517943062000,
      "permanentid" : "0f38a9b49ce333e372259db1f8c1adfbb134588282db6da7d37370c39985",
      "syslanguage" : [ "English" ],
      "transactionid" : 160364,
      "title" : "\"Request Error : Error while performing JWT flow for the requested user\"",
      "date" : 1490227200000,
      "objecttype" : "Answers",
      "sitemapchangefrequency" : "Monthly",
      "hasanswer" : "false",
      "audience" : [ "Administrator", "Developer" ],
      "commentauthor" : [ "gminero", "Martin Laporte \u2666\u2666", "laurent", "gminero", "laurent" ],
      "hasacceptedanswer" : "false",
      "sourcetype" : "Sitemap",
      "sysconnectortype" : "SitemapCrawler",
      "rowid" : 5448833,
      "numberofcomments" : 3,
      "size" : 75766,
      "sysdocumenttype" : "WebPage",
      "questionvotes" : 0,
      "clickableuri" : "https://answers.coveo.com/questions/10855/request-error-error-while-performing-jwt-flow-for.html",
      "syssource" : "AnswersCloud",
      "orderingid" : 212384744911579113,
      "acceptedanswers" : 0,
      "syssize" : 75766,
      "sysdate" : 1490227200000,
      "author" : "laurent",
      "source" : "AnswersCloud",
      "questionbodyhtml" : "<div class=\"question-body\">\n                <p>\n<p>I'm getting this error when accessing my community as a sysadmin user:</p>\n<pre>{\n  \"message\": \"Request Error : Error while performing JWT flow for the requested user.\",\n  \"status\": 400,\n  \"type\": \"Ajax Error (status: 400)\",\n  \"name\": \"Ajax Error (status: 400)\"\n}\n</pre>\n<p>I've already looked at the 2 following questions:</p>\n<ul> \n <li><a rel=\"nofollow\" href=\"https://answers.coveo.com/questions/10270/error-while-performing-jwt-flow-for-the-requested.html\">https://answers.coveo.com/questions/10270/error-while-performing-jwt-flow-for-the-requested.html</a></li>\n <li><a rel=\"nofollow\" href=\"https://answers.coveo.com/questions/10523/error-configuring-a-search-page.html\">https://answers.coveo.com/questions/10523/error-configuring-a-search-page.html</a></li>\n</ul>\n<p>However the suggested answers don't help: I've already made sure that the needed profiles/permission sets where correctly added to the \"Coveo for Salesforce\" connected app.</p>\n<p>I've then tried to disable the \"fallback on admin identity\" option: although this does allow the sysadmin to finally see search results, this doesn't work for external authenticated users: they just see a blank search results page (the same one they see when the \"fallback on admin identity\" option is enabled).</p> \n<p>\n                </div>",
      "sitemaplastmodified" : 1490227200000,
      "collection" : "default",
      "syssourcetype" : "Sitemap",
      "indexeddate" : 1517943062000,
      "questionnumber" : 10855,
      "connectortype" : "SitemapCrawler",
      "filetype" : "html",
      "sysclickableuri" : "https://answers.coveo.com/questions/10855/request-error-error-while-performing-jwt-flow-for.html",
      "sysfiletype" : "html",
      "language" : [ "English" ],
      "sitemapparenturl" : "http://answers.coveo.com/questions/1.xml",
      "questionauthor" : "laurent",
      "sitename" : "Answers",
      "sysrowid" : 5448833,
      "uri" : "https://answers.coveo.com/questions/10855/request-error-error-while-performing-jwt-flow-for.html",
      "syscollection" : "default",
      "sitelanguage" : [ "English" ]
    },
    "Title" : "\"Request Error : Error while performing JWT flow for the requested user\"",
    "Uri" : "http://answers.coveo.com/questions/10855/request-error-error-while-performing-jwt-flow-for.html",
    "PrintableUri" : "https://answers.coveo.com/questions/10855/request-error-error-while-performing-jwt-flow-for.html",
    "ClickUri" : "https://answers.coveo.com/questions/10855/request-error-error-while-performing-jwt-flow-for.html",
    "UniqueId" : "42.4247$http://answers.coveo.com/questions/10855/request-error-error-while-performing-jwt-flow-for.html",
    "Excerpt" : "I'm getting this error when accessing my community as a sysadmin user: ... I've already looked at the 2 following questions: ... However the suggested answers don't help: I've already made sure tha...",
    "FirstSentences" : null
  }